Gems/Atom/RHI/Vulkan/Code/Source/RHI/Device.h

```diff
@@ -214,6 +224,11 @@ namespace AZ
 
             BindlessDescriptorPool m_bindlessDescriptorPool;
             ShadingRateImageMode m_imageShadingRateMode = ShadingRateImageMode::None;
+
+            RHI::ResultCode InitVulkanAllocator(RHI::PhysicalDevice& physicalDevice);
+            void ShutdownVulkanAllocator();
+
+            VmaAllocator m_vmaMemoryAllocator;
```

Comment:
`VmaAllocator` can be made a pointer and forward declare here.
That allow `vk_mem_alloc.h` to only be included in the cpp file

Reply:
VmaAllocator is already a typedef of a ptr. I just recreated the typedef at the top of the file and removed the include of vk_mem_alloc.h.